### PR TITLE
Update (Github/PR) pull_request_template.md

### PR DESCRIPTION
**Pull request by pangolp**
### Description

The structure of the file has been modified to add links to Spanish and a few words of thanks to the contributors. Also, a message is left, that if you need to make an update in the page or creation in Spanish of the same, please tag @pangolp so he can make such update.

### Related Issue

Closes

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
+- 
+- 
+
 ### Related Issue
 
 Closes
+
+## Thank you for contributing to the AzerothCore wiki.
+
+If you are making a change to a file that requires an update to Spanish or you are creating one, please, if you can, within the pull request or the chat itself, tag or mention **@pangolp** so that he can create/update the file to Spanish as well. Thank you.
```
Comment (Kitzunu) [inline PULL_REQUEST_TEMPLATE.md:19]: I'd suggest making this a comment or you will be spammed af :P

Comment (Kitzunu) [reply PULL_REQUEST_TEMPLATE.md:19]: Also technically it would apply for any non Spanish PR

Comment (pangolp) [reply PULL_REQUEST_TEMPLATE.md:19]: We'll put it as a comment then.

**Comment by pangolp** [comment]
That way, or the links too?

**Comment by pangolp** [comment]
@Helias if you can approve this, so we make the merge, thanks.

**Review by Helias — APPROVED** [review]

**Comment by pangolp** [comment]
Thanks @Helias

**Pull request merged**